Description: Make sure vapi workspace def use real c header filenames
Author: fossfreedom <fossfreedom@ubuntu.com>
Commit: https://github.com/UbuntuBudgie/budgie-desktop/commit/825353ea27af8ca2e54e95421072561a1bcfb488
Last Update: 2018-08-26

---
 vapi/libmutter-3.vapi | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/vapi/libmutter-3.vapi b/vapi/libmutter-3.vapi
index 8a5f5ab0..bd8cc632 100644
--- a/vapi/libmutter-3.vapi
+++ b/vapi/libmutter-3.vapi
@@ -708,7 +708,7 @@ namespace Meta {
 		public Cairo.Region to_region (int center_width, int center_height);
 		public void unref ();
 	}
-	[CCode (cheader_filename = "meta/main.h", type_id = "meta_workspace_get_type ()")]
+	[CCode (cheader_filename = "meta/workspace.h", type_id = "meta_workspace_get_type ()")]
 	public class Workspace : GLib.Object {
 		[CCode (has_construct_function = false)]
 		protected Workspace ();
@@ -728,7 +728,7 @@ namespace Meta {
 		public signal void window_added (Meta.Window object);
 		public signal void window_removed (Meta.Window object);
 	}
-	[CCode (cheader_filename = "meta/main.h", type_id = "meta_workspace_manager_get_type ()")]
+	[CCode (cheader_filename = "meta/meta-workspace-manager.h", type_id = "meta_workspace_manager_get_type ()")]
 	public class WorkspaceManager : GLib.Object {
 		[CCode (has_construct_function = false)]
 		protected WorkspaceManager ();
--
2.17.1

